packages/botonic-react/src/components/index-types.ts

```diff
@@ -151,9 +152,12 @@ export interface ThemeProps extends StyleProp {
     enableArrows?: boolean
   }
   reply?: StyleProp & CustomProp
-  triggerButton?: { notificationsEnabled?: boolean } & ImageProp &
-    StyleProp &
-    CustomProp
+  triggerButton?: ImageProp & StyleProp & CustomProp
+  notifications?: EnableProp & {
```

Comment:
At the moment it doesn't work like that but I think it would be the right way. In the description of the PR in the explanation of the third commit I put  (TODO: maybe it should also be activated by default when activating notifications in general).

Reply:
I have refactored this so that setting notifications: { enable: true } also enables notifications in the trigger button.

Reply:
Applied